ansible/configs/ocp-workloads/examples/cakephp-openshift-applier.yml

```diff
@@ -0,0 +1,38 @@
+---
+cloud_provider: none
+env_type: ocp-workloads
+
+ocp_workloads:
+  - ocp-workload-openshift-applier
+
+ocp_username: gucore-redhat.com
+guid: testgucore
+
+openshift_cluster_content:
+
+  - object: projectrequest
+    content:
+    - name: Create namespace
+      template: https://raw.githubusercontent.com/redhat-cop/openshift-applier/v2.1.1/tests/files/templates/projectrequest.yml
+      params_from_vars:
+        NAMESPACE: "{{ guid }}"
+        NAMESPACE_DESCRIPTION: Sample namespace for student {{ ocp_username }}
+        NAMESPACE_DISPLAY_NAME: Sample namespace for student {{ ocp_username }}
+      action: "{{ applier_action }}"
+
+  - object: cakephprequest
+    content:
+    - name: Create cake-php mysql
+      template: https://raw.githubusercontent.com/openshift/origin/v4.1.0/examples/quickstarts/cakephp-mysql.json
+      params_from_vars:
+        NAME: "cakephp-mysql-{{ guid }}"
+      namespace: "{{ guid }}"
+      action: "{{ applier_action }}"
+
+target_host:
+  ansible_host: bastion.dev.openshift.opentlc.com
+  ansible_port: 22
+  ansible_user: ec2-user
+  #ansible_ssh_private_key_content: "{{ target_host_ssh_private_key_content }}"
+  ansible_ssh_private_key_file: ~/.ssh/opentlc_admin_backdoor.pem
```

Comment:
This is not going to work on tower, right? Unless we copy the key to every tower node?
Do you want to merge as-is and then solve the SSH key issue in another place?

Reply:
private key *file* won't but private key *content* will. So we will use something like:

```yaml
ansible_ssh_private_key_content: "{{ target_host_ssh_private_key_content }}"
```

And set the variable in a secret.

In the ocp-workloads config, there is a step that create the key in output_dir, and use it to connect to the bastion. Then it's cleaned up.